Update index.mdx (#2132)
hashicorp/terraform-website: content/enterprise/before-installing/index.mdx

#### Instance Profile as Default Credentials

> You can use Terraform Enterprise's instance profile to provide default credentials to workspaces. Configure the [instance metadata options](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-options.html) and specify the PUT response hop limit value as 2 for instances using IMDSv2. Terraform will attempt to use the instance profile to provision resources when you do not set credentials as environment variables. However, this approach presents a few security risks:

Reviewer: ```suggestion
You can use Terraform Enterprise's instance profile to provide default credentials to workspaces. When using IMDSv2, configure the PUT response hop limit with a value of 2 within the [instance metadata options](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-options.html). Terraform will attempt to use the instance profile to provision resources when you do not set credentials as environment variables. However, this approach presents a few security risks:
```
